src/components/MissingDataPopup/MissingDataPopup.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useState } from 'react';
+import styles from './MissingDataPopup.module.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PathwayPopup from '../PathwayPopup/PathwayPopup';
+import nodeClasses from '../ExpandedNode/ExpandedNode.module.scss';
+import ActionButton from '../ActionButton/ActionButton';
```

Comment:
Same thing here - these should be absolute not relative imports and the scss module should be imported as `nodeStyles`